pkg/gardenlet/controller/managedseed/managedseed_controller.go

```diff
@@ -31,9 +31,30 @@ func (c *Controller) managedSeedAdd(obj interface{}) {
 		return
 	}
 
-	if managedSeed.Generation != managedSeed.Status.ObservedGeneration || managedSeed.DeletionTimestamp != nil {
+	generationChanged := managedSeed.Generation != managedSeed.Status.ObservedGeneration
+	jitterUpdates := (c.config.Controllers.ManagedSeed.JitterUpdates != nil) && *(c.config.Controllers.ManagedSeed.JitterUpdates)
+	if managedSeed.DeletionTimestamp != nil {
 		c.logger.Debugf("Added ManagedSeed %s without delay to the queue", key)
 		c.managedSeedQueue.Add(key)
+		return
+	}
+
+	// Newly created managed seed will be enqueued immediatly. Generation is 1 for newly created objects.
```

Comment:
```suggestion
	// Newly created managed seed will be enqueued immediately. Generation is 1 for newly created objects.
```

Reply:
Addressed in [commit](https://github.com/gardener/gardener/pull/5483/commits/de284d6ecaa02c1411dda2dee6accfbb949d97f5)